port/win/env_win.cc

```diff
@@ -1637,6 +1637,7 @@ class WinEnv : public Env {
     return s;
   }
 
+#if (_WIN32_WINNT >= _WIN32_WINNT_WIN8)
```

Comment:
This ifdef makes the behavior of the code dependent on the build system not on the runtime environment. It much more expressive to make it depend on the `#define` which is driven by CMake switch which would explicitly express the user wish.